src/scss/helpers/_contains.scss

```diff
@@ -0,0 +1,29 @@
+@charset "UTF-8";
+
+// Checks if a list contains a value(s).
+//
+// @param {list} $list
+//   The list to check against.
+//
+// @param {list} $values
+//   A single value or list of values to check for.
+//
+// @example scss - Usage
+//   contains($list, $value)
+//
+// @return {bool}
+// - - - - - - - - - - - - - - - - - - - - - - - - -
+@function contains($list, $values...) {
+  @if not is-list($list) {
+      @warn 'argument error: #{$list}';
+      @return 'error';
```

Comment:
Line should be indented 4 spaces, but was indented 6 spaces<br>Prefer double-quoted strings